recipes/dtw/LICENSE

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
You ship here version 3.0 but you indicate below that it is version 2.

Reply:
GPL-2.0-or-later should be fine or?

Reply:
But this file, the LICENSE file, states `Version 3` is that correct?

Reply:
Isn't GPL-2.0-or-later meaning GRL >=2 which includes Version 3?

Reply:
Its still GPL-2.0 or later, so I guess this one should be included: https://www.gnu.org/licenses/old-licenses/gpl-2.0-standalone.html

Reply:
If this project ships GPL3 why is then the license in the field GPL-2-or-later and not GPL-3-only or GPL-3-or later https://github.com/conda-forge/staged-recipes/pull/22951/files#diff-e20acf87e2351283380a6dc7b5d2a955b8d1a0f768aa16b4e6c3e5fc7d3d71d9R36

Both exists: https://spdx.org/licenses/GPL-3.0-only.html https://spdx.org/licenses/GPL-3.0-or-later.html